### ISSUE TITLE
Readmanga http error 415

### ISSUE THREAD

**Issue by supremko** [tachiyomiorg/tachiyomi-extensions#14674]
### Source information and language

Readmanga 1.3.55 (Русский)

### Steps to reproduce

Some mangas give error 415 when try to download or just doesn't open when try read without downloading.
As example https://readmanga.live/nachalo_posle_konca__A5664
In webview opens without problem

### Expected behavior

Download without errors

### Actual behavior

Give error 415

### Tachiyomi version

0.14.2

### Android version

Android 11

### Other details

After #14656 http error 300 gone, but 415 appeared

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open or closed issue.
- [X] I have written a short but informative title.
- [X] I have updated the app to version **[0.14.2](https://github.com/tachiyomiorg/tachiyomi/releases/latest)**.
- [X] I have updated all installed extensions.
- [X] I have tried the [troubleshooting guide](https://tachiyomi.org/help/guides/troubleshooting/).
- [X] If this is an issue with the app itself, I should be opening an issue in the [app repository](https://github.com/tachiyomiorg/tachiyomi/issues/new/choose).
- [X] I will fill out all of the requested information in this form.

**Comment by slehernaf** [comment]
I have problem with open and download, and have 502 error, and this some sad🥲...

**Comment by supremko** [comment]
> I have problem with open and download, and have 502 error, and this some sad🥲...

Yeah, found manga that give 502 too, but in webview all ok. As example https://readmanga.live/prirojdennyi_naemnik__A5664 (first page ok, second and other 502)

**Comment by slehernaf** [comment]
I don't have the first, but also one or two pages in the chapter can be displayed normally (or maybe none), while others with the error "failed to load the image".
For example, [here ](https://readmanga.live/pik_boevyh_iskusstv/vol1/2891)2 pages are loaded, and [here ](https://readmanga.live/pik_boevyh_iskusstv/vol1/2894)0.
